src/components/MobileMenu.svelte

```diff
@@ -1,14 +1,18 @@
+<script>
+  import NavLinks from "./NavLinks.svelte";
+</script>
+
 <style>
-  #mobile-menu-list li {
+  :global(#mobile-menu-list li) {
```

Comment:
Using `:global` selector seems a rather hacky way to work around Svelte's compiler. Can we avoid it by merging the nav components together? This is how I imagine doing it:
- A `header` element containing:
    - the hamburger menu `input` & `span`s
    - the logo `img`
    - a `nav` element
        - the links `ul`
        - the action button as a separate `a` element
- On desktop
    - hamburger menu is hidden
- On mobile
    - hamburger menu is shown
    - links `ul` is switched to `position: fixed` and respond to state of `input`.

Most likely these changes is non-trivial, so we can hold it and see if we have time to do it.